backend/src/libs/packages/server-application/libs/constants/api.constants.ts

```diff
@@ -0,0 +1,9 @@
+import { APIPath } from '#libs/enums/enums.js';
+import { AuthApiPath } from '#packages/auth/libs/enums/enums.js';
+
+const WHITE_ROUTES = [
+  { path: `/api/v1${APIPath.AUTH}${AuthApiPath.SIGN_UP}`, method: 'POST' },
```

Comment:
Let's use array here
```suggestion
  { path: `/api/v1${APIPath.AUTH}${AuthApiPath.SIGN_UP}`, methods: ['POST'] },
```